proc_IR_noecho.py

```diff
@@ -115,6 +115,7 @@ def vec_lcurve(l):
     sfo1 = 272.05
     arbitrary_reference = s.get_prop('acq')['BF1'] 
     logger.info(strm("SFO1 is",sfo1))
+    s.ft('t2')
     s.setaxis('t2',lambda x:x + sfo1 - arbitrary_reference)
```

Comment:
line 119 here is clearly adjusting the frequency axis, so you need to be in the Frequency domain for this to make sense